src/components/Accordion.js

```diff
@@ -0,0 +1,93 @@
+const menuOpen = "collapse show";
+const buttonOpenAll = "Open All";
+const buttonCloseAll = "Close All";
+
+if (!String.prototype.includes) {
+  String.prototype.includes = function(search, start) {
+    "use strict";
+    if (typeof start !== "number") {
+      start = 0;
+    }
+
+    if (start + search.length > this.length) {
+      return false;
+    } else {
+      return this.indexOf(search, start) !== -1;
+    }
+  };
+}
+
+document.addEventListener(
+  "click",
+  clickEvent => {
+    const { target } = clickEvent;
+
+    if (target && target.className === "dg_allitems") {
+      AllMenuItemsAction(target);
+    } else if (target) {
+      MenuAction(target);
+    } else {
+      return;
+    }
+  },
+  false
+);
+
+function MenuAction(element) {
+  var mainDiv = element.parentElement.parentElement;
+  var menuItems = mainDiv.getElementsByClassName("multi-collapse");
+  var childDiv = element.nextElementSibling;
+  var menuID = childDiv.id;
+
+  for (i = 0; i < menuItems.length; i++) {
+    if (menuID !== menuItems[i].id) {
+      menuItems[i].className = mainDiv.className.includes(
+        "dg_allowmutlipleopen"
+      )
+        ? menuItems[i].className
+        : menuState("close");
+    } else {
+      if (menuItems[i].className.includes(menuOpen)) {
```

Comment:
you could simplify some of this logic with ternary operator

example

```js
const isMenuOpen = menuItem.className.includes(menuOpen);
menuItem.setName = menuState(isMenuOpen ? "close" : "open");
...
```

Reply:
yes, but this is a refactor and needs to wait until we are refactoring the code. Timelines and all.

Reply:
?

Reply:
Code refactoring is supposed to be after the project and during the refactor. This is working code and it would be nice to have a ternary instead of if then but can wait.

Reply:
I don't agree with that, but not blocking.